src/_pytest/nodes.py

```diff
@@ -49,7 +49,7 @@ def _splitnode(nodeid):
         []
         ['testing', 'code']
         ['testing', 'code', 'test_excinfo.py']
-        ['testing', 'code', 'test_excinfo.py', 'TestFormattedExcinfo', '()']
+        ['testing', 'code', 'test_excinfo.py', 'TestFormattedExcinfo']
```

Comment:
would it make sense to turn this one into a doctest?

Reply:
Thought the same initially, but we should rather have proper tests for it in the long run (https://github.com/pytest-dev/pytest/pull/6394#issuecomment-574138866).

Reply:
In the long run I want it anhillated